packages/lib/BaseApplication.ts

```diff
@@ -927,6 +929,36 @@ export default class BaseApplication {
 
 		await MigrationService.instance().run();
 
+		this.startUpRotateLogFilesRoutine(profileDir);
+
 		return argv;
 	}
+
+	private startUpRotateLogFilesRoutine(profileDir: string) {
+		const rl: RotationalLogs = new RotatingLogs(profileDir);
+		this.cleanActiveLogWrapper(rl);
+		this.deleteNonActiveLogFilesWrapper(rl);
+	}
+
+	private cleanActiveLogWrapper(rl: RotationalLogs) {
+		const sixtySeconds: number = 60 * 1000;
+		shim.setTimeout(async () => {
+			try {
+				await rl.cleanActiveLogFile();
+			} catch (e) {
+				appLogger.error('Cannot clean the active log file.', e);
+			}
+		}, sixtySeconds);
```

Comment:
It's ok to just plain numbers here, since developers know that 60 * 1000 is 60 seconds.